### PR TITLE
update dockerfile and remove histomictk

### PR DESCRIPTION
**Pull request by sayatmimar**
This PR updates the Dockerfile (base image and dependencies) and `histomicstk` folders.

### CODE REVIEW

**Comment by sayatmimar** [comment]
Yes so David confirmed that we do not need the `histomicstk` library and related folders so I removed them. I also renamed the package directory that why number of changed files is high in the PR.

File: setup.py [hunk 1]
```diff
@@ -76,21 +66,21 @@ def prerelease_local_scheme(version):
         'tifffile==2023.4.12',
         'tiffslide==1.5.0',
```
Comment (anindya-paul) [inline setup.py:67]: Can we update to latest tiffslide?

**Review by anindya-paul — APPROVED** [review]
Minor comments, overall looks good.

File: Dockerfile [hunk 2]
```diff
@@ -108,73 +108,26 @@ ENV build_path=$PWD/build
...
-    pip install --no-cache-dir 'large-image[memcached]' && \
-    # Install HistomicsTK \
-    pip install --no-cache-dir . --find-links https://girder.github.io/large_image_wheels && \
-    # Install tf-slim \
-    pip install --no-cache-dir 'tf-slim>=1.1.0' && \
-    # Install pillow_lut \
-    pip install --no-cache-dir 'pillow-lut' && \
-
+    pip install --no-cache-dir .  && \
     pip install --no-cache-dir tensorboard cmake onnx && \
```
[the remark this answers is not in the document]
Comment (sayatmimar) [reply Dockerfile:123]: Detectron2 depends on those that's why I added them.

**Pull request merged**